test/AccountManager.test.ts

```diff
@@ -99,38 +99,37 @@ describe('AccountManager', function () {
     });
 
     describe('removeAccount()', function () {
-      let fakeAccounts: Array<Wallet>;
+      let randomAccounts: Array<Wallet>;
+      let initialLength: number;
 
       beforeEach(function () {
-        fakeAccounts = new Array<Wallet>(3).fill(Wallet.createRandom());
+        initialLength = 3;
```

Comment:
I'd store this in a constant, no need to initialize it each time in a beforeEach

Reply:
Done.